telegram/ext/updater.py

```diff
@@ -172,7 +190,7 @@ def __init__(
                     request_kwargs = {}
                 if 'con_pool_size' not in request_kwargs:
                     request_kwargs['con_pool_size'] = con_pool_size
-                self._request = Request(**request_kwargs)
+                self.__class__._request = Request(**request_kwargs)
```

Comment:
Oh it is defined as a class variable, and then initialized as an instance variable (not sure if that was intended) in the `__init__` so I couldn't add `_request` to `__slots__` since `_request` is a class variable too (it raises an error).

Reply:
I read up on that a bit …

```python
class Foo:
    foo = 42

f = Foo()
f.foo = 7
print(Foo.foo)  # 42
print(f.foo)  # 7
print(f.__class__.foo)  # 42
```

`f.foo` *shadows* `Foo.foo`. Here it should be as easy as moving the `_request=None` to the `__init__`

Reply:
Yep, that was basically what I meant. Wanted to make sure there would be unintended side effects of moving _request to __init__